### DISCUSSION TITLE
Aggiunta in Developers Italia

### DISCUSSION THREAD

**Discussion by bfabio** [iisgiua/giuaschool#253] [Richieste di aiuto] [answered]
Ciao e complimenti per il progetto :pray: 

sarebbe bello inserire giuaschool nel catalogo di Developers Italia (https://developers.italia.it/it/riuso/pubblicazione), per questo vi chiedo: la [titolarità del software](https://docs.italia.it/italia/developers-italia/lg-acquisizione-e-riuso-software-per-pa-docs/it/stabile/premessa/titolarita.html) è della scuola?

Questo ci serve per capire se pubblicare come Pubblica amministrazione o [software di terze parti](https://developers.italia.it/it/riuso/pubblicazione#software-open-source-di-terze-parti)

Altra domanda: siete in contatto con chi sviluppa https://github.com/Giua-app e/o avete in programma di unire i progetti?

Grazie!

**Comment by trinko** [comment, marked as answer]
Si, il software è  a pieno titolo dell'I.I.S. Michele Giua, in quanto realizzato da risorse interne all'amministrazione.
Il progetto Giua-App è realizzato da alcuni nostri studenti, ma è un progetto separato, gestito in autonomia da loro.

**Comment by bfabio** [comment]
@trinko grazie!

Pensi sia fattibile fare un'organizzazione GitHub per la scuola, spostare lì giuaschool (ed eventuale altro software della scuola) e fare la [procedura di onboarding sul catalogo](https://onboarding.developers.italia.it/)?

**Reply by trinko** [reply]
> Pensi sia fattibile fare un'organizzazione GitHub per la scuola, spostare lì giuaschool (ed eventuale altro software della scuola) e fare la [procedura di onboarding sul catalogo](https://onboarding.developers.italia.it/)?

In questo momento no, mi dispiace. In seguito vedremo.

**Reply by bfabio** [reply]
@trinko anche se è preferibile l'onboarding ufficiale, possiamo [sempre farlo manualmente](https://github.com/italia/developers-italia-backend/blob/master/crawler/whitelist/manual-reuse.yml)

**Reply by trinko** [reply]
Ok. Grazie
